lib/active_model_serializers.rb

```diff
@@ -18,11 +18,13 @@ def silence_warnings
 require 'active_model/serializer/version'
 
 require 'action_controller/serialization'
+require 'action_controller/serialization_test_case'
 ActiveSupport.on_load(:action_controller) do
   include ::ActionController::Serialization
   ActionDispatch::Reloader.to_prepare do
     ActiveModel::Serializer.serializers_cache.clear
   end
+  ActionController::TestCase.send(:include, ::ActionController::SerializationAssertions)
```

Comment:
I'd rather put something like this in `active_model_serializers/test.rb`, no? like

- https://github.com/rails/rails/blob/master/activejob/lib/active_job/test_case.rb

```ruby
require 'active_support/test_case'

module ActiveJob
  class TestCase < ActiveSupport::TestCase
    include ActiveJob::TestHelper
  end
end
```

- https://github.com/rails/rails/blob/3e36db4406beea32772b1db1e9a16cc1e8aea14c/activejob/lib/active_job.rb#L35-L36

```ruby
module ActiveJob
  extend ActiveSupport::Autoload

  autoload :TestCase
  autoload :TestHelper
end
```
then

```ruby
class ActiveJobTestCaseTest < ActiveJob::TestCase
```
- https://github.com/rails/rails/blob/master/activejob/lib/active_job/test_helper.rb